lib/pages/dashboard/dashboard_club_teams.dart

```diff
@@ -59,6 +59,7 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
 
   @override
   Widget build(BuildContext context) {
+    super.build(context);
```

Comment:
Just for my knowledge, did this line solve the performance issue?

Reply:
Yes it has solved the performance issue !